README.adoc

```diff
@@ -84,6 +84,16 @@ https://tapestry.apache.org/[Tapestry 5] IoC container.
 
 === Building
 
+==== Prerequisites
+
+Spock needs both a JDK 8 and JDK 11+ installed.
```

Comment:
I understand on CI, but why just JDK 11+ wouldn't be enough to build a Spock version locally?

Reply:
Because Spock uses Toolchains to compile core with Java 8, there are subtle differences to using `--release 8` and I think it is best that local and CI behaves the same.

Reply:
Nowadays, not all people (potential Spock contributors) have JDK 8 installed. Nevertheless, with SDKMAN or any other way it should be doable, so probably it is not a big problem (probably also for M1 Mac users :-) ).